crawler/src/main/java/com/brandwatch/minibcr/crawler/services/CrawlerService.java

```diff
@@ -0,0 +1,30 @@
+package com.brandwatch.minibcr.crawler.services;
+
+
+import org.springframework.stereotype.Service;
+
+import com.github.redouane59.twitter.TwitterClient;
+import com.github.redouane59.twitter.dto.tweet.Tweet;
+
+import com.brandwatch.minibcr.common.domain.Resource;
+
+@Service
+public class CrawlerService {
+    private static final TwitterClient twitterClient = new TwitterClient();
```

Comment:
How do you configure the client?

Reply:
I suggest you have a configuration class where you can configure the client and make it as a bean, so after that you can inject it in the CrawlerService. You can also set the values for it in application.properties file, instead of using json configuration file and like that you will have everything in once place.